Formula/breakpad.rb

```diff
@@ -0,0 +1,69 @@
+class Breakpad < Formula
+  desc "Components implement a crash-reporting system."
+  homepage "https://chromium.googlesource.com/breakpad/breakpad/"
+
+  url "https://chromium.googlesource.com/breakpad/breakpad",
+      :using => :git,
+      :revision => "67d5567177301d0c24303f26ad119ab7bd7fab40"
```

Comment:
I'm not sure whether everybody use tag to manage its version, even I can submit an issue to them, I doubt Google will change its R&D workflow to follow up yours.

On the other hand, I notice there also have some formulas without an official tag, for example

https://github.com/Homebrew/homebrew-core/blob/master/Formula/x264.rb